### PR TITLE
Add chatwork

### PR DESCRIPTION
**Pull request by magcho**

### CODE REVIEW

File: entries/c/chatwork.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "img": "chatwork.com.png",
+    "tfa": [
+      "sms",
+      "totp",
+    ],
+    "keywords": [
+      "communication",
+    ]
+  }
+}
```
Comment (40504) [inline entries/c/chatwork.com.json:15]:
```json
{
  "Chatwork": {
    "domain": "chatwork.com",
    "url": "https://go.chatwork.com/",
    "tfa": [
      "sms",
      "totp"
    ],
    "documentation": "https://support-en.chatwork.com/hc/en-us/articles/360023097632",
    "keywords": [
      "communication"
    ]
  }
}
```

**Review by 40504 — CHANGES_REQUESTED** [review]
minified and added documentation URL, also removed unnecessary commas that make JSON file invalid

File: entries/c/chatwork.com.json (hunk 1 above)
Comment (magcho) [reply entries/c/chatwork.com.json:15]: Thank you very much ! (⌒▽⌒)

File: entries/c/chatwork.com.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "Chatwork": {
+    "domain": "chatwork.com",
+    "url": "https://go.chatwork.com/",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support-en.chatwork.com/hc/en-us/articles/360023097632",
```
Comment (ApCoder123) [inline entries/c/chatwork.com.json:9]:
```suggestion
    "documentation": "https://support-en.chatwork.com/hc/en-us/articles/360023097632",
    "notes": "SMS 2FA is only available to users on paid plans",
```

According to [this page](https://support-en.chatwork.com/hc/en-us/articles/360023097452):

> * Two-factor authentication using SMS cannot be used with the free plan. Please use two-factor authentication using the authentication app.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: entries/c/chatwork.com.json (hunk 2 above)
Comment (magcho) [reply entries/c/chatwork.com.json:9]: I overlooked. Thank you 👍

**Review by ApCoder123 — APPROVED** [review]

**Review by 40504 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**